data/fro/VanEmden_1977_GirViane/metadata.txt

```diff
@@ -0,0 +1,25 @@
+publicationType: "book"
+publicationTitle: "Girart de Vienne par Bertrand de Bar-sur-Aube"
+publicationDate: "1977"
+publicationPlace: "Paris"
+publicationSeries: "SATF"
+publicationNum: "94"
+publicationStemmaNum: ""
+publicationAuthors: "Van Emden, Wolfgang"
+publicationPage: "LXIII"
+publicationLink: ""
+workTitle: "Girart de Vienne"
+workViaf: "2146522337332391758"
+workOrigDate: ""
+workOrigPlace: ""
+workAuthor: "Bertrant de Bar-sur-Aube"
+workAuthorViaf: "61541851"
+workGenre: "chanson de geste"
+workLangCode: "fro"
+stemmaType: "reconstructed"
+contam: "yes"
```

Comment:
Is there contamination within the stemma?

Reply:
Not in the plotted stemma itself, but the text says that ms. S switches family at the beginning and end of the text.